packages/contracts-bedrock/src/L1/SystemConfig.sol

```diff
@@ -539,4 +539,10 @@ contract SystemConfig is ProxyAdminOwnedBase, OwnableUpgradeable, Reinitializabl
     function guardian() public view returns (address) {
         return superchainConfig.guardian();
     }
+
+    /// @notice Returns whether the custom gas token feature is enabled.
+    /// @return bool True if the custom gas token feature is enabled, false otherwise.
+    function isCustomGasToken() public view returns (bool) {
```

Comment:
Feels unnecessary, is there a particular reason to have this?

Reply:
We should add the `/// @custom:legacy` tag then and remove it in a future release